src/Storage/Query/Handler/PagingHandler.php

```diff
@@ -22,20 +22,20 @@ public function __invoke(ContentQueryParser $contentQuery)
 
         foreach ($contentQuery->getContentTypes() as $contenttype) {
             //Find out if we are searching or just doing a simple query
-            if ($searchParam = $contentQuery->getParameter('filter')) {
+            if ($contentQuery->hasParameter('filter')) {
```

Comment:
This part breaks in Bolt 3.2:

> Attempted to call an undefined method named "hasParameter" of class "Bolt\Storage\Query\ContentQueryParser".
> Did you mean to call e.g. "getParameter", "getParameters", "setParameter" or "setParameters"?

Reproducable with Bolt 3.2 + some Lorem Ipsum content
and navigate to `/json/pages`.